### ISSUE TITLE
[REQ] Move finished plot file instead of copying when temp and final dirs are on the same drive

### ISSUE THREAD

**Issue by kilrah** [Chia-Network/chia-blockchain#3932]
Currently once plotting is finished the final big .tmp file is copied to the destination in all cases and the source deleted, even when the destination and temporary folders are on the same drive. A move operation when that is the case would avoid the time and wear of writing an entire file again for nothing.

**Comment by olafbuitelaar** [comment]
I think this is a duplicate of this; https://github.com/Chia-Network/chia-blockchain/issues/1868

**Comment by chiabug** [comment]
As an addition to this it would be helpful if there was a parameter that allowed the final plot to not be copied or moved at all.  The process would simply rename the end plot to a .plot and leave it alone.  This would be helpful to offline plotters.  This would generally be used in conjunction with the -x parameter so perhaps -x2 could mean don't insert the the plot into the config and don't move the plot anywhere after creation.  I can submit this as a suggested enhancement as well.

**Comment by x3style** [comment]
Yes please, i plot in the same drive and it literally takes an extra few hours on something that could very well be 1 second.
